addons/website_blog/tests/test_menu.py

```diff
@@ -0,0 +1,35 @@
+import odoo.tests
```

Comment:
`# Part of Odoo. See LICENSE file for full copyright and licensing details.`
not really useful for tests but.. might just put it everywhere
utf-8 can be omitted tho

FYI: https://discord.com/channels/678381219515465750/687338482598674454/1142067066090815528
> it's best to add it because it's always better to make the license explicit, if anyone happens to stumble upon the file anywhere out of context. It doesn't mean that the license or copyright doesn't apply if we don't do it - it just makes it clear and explicit.